crypto/cms/cms_kari.c

```diff
@@ -385,9 +385,6 @@ int cms_RecipientInfo_kari_init(CMS_RecipientInfo *ri,  X509 *recip, EVP_PKEY *r
          /* Use originator key */
          CMS_OriginatorIdentifierOrKey *oik = ri->d.kari->originator;
 
-         if (originatorPrivKey == NULL && originator == NULL)
```

Comment:
Shouldn't it be actually kept and just `&&` changed to `||`?

Reply:
I'm not sure: CMS isn't something I know much about.  Except to know that I don't want to know more if I can help if :)  Still, it seems reasonable since these are dereferenced without checking.

The indentation was off by one here too.